NOTICE

```diff
@@ -5,7 +5,7 @@
    =========================================================================
 Progress Spark Toolkit Demos
 
-Copyright © 2020 Progress Software Corporation and/or one
+Copyright © 2021 Progress Software Corporation and/or one
```

Comment:
Also update

Reply:
No, it was updated then and not since. Just need it pushed out